keys/server_test.go

```diff
@@ -37,7 +38,7 @@ var (
 func init() {
 	failedCh := make(chan error)
 	go func() {
-		err := StartStandAloneServer(DefaultKeysDir, DefaultHost, TestPort)
+		err := StartStandAloneServer("test_scratch/"+DefaultKeysDir, DefaultHost, TestPort, false, logging.NewNoopLogger())
```

Comment:
Oh let's also `os.RemoveAll(testDir)` at beginning of test, ala: https://github.com/hyperledger/burrow/blob/develop/core/integration/test_wrapper.go#L54-L56